src/us/kbase/workspace/test/kbase/TypeDelegationTest.java

```diff
@@ -112,8 +133,9 @@ public static void setUp() throws Exception {
 		MONGO = new MongoController(TestCommon.getMongoExe(),
 				Paths.get(TestCommon.getTempDir()),
 				TestCommon.useWiredTigerEngine());
-		System.out.println("Using mongo temp dir " + MONGO.getTempDir());
 		final String mongohost = "localhost:" + MONGO.getServerPort();
+		System.out.println(String.format("Started monngo at %s with temp dir %s",
```

Comment:
typo in monnnnnnngo

Reply:
Looks like I already fixed that somewhere between the current commit and the tip of my local develop branch